GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.js

```diff
@@ -479,7 +479,45 @@ gdjs.RuntimeGamePixiRenderer.prototype.openURL = function (url) {
     Cocoon.App.openURL(url);
   } else if (typeof window !== 'undefined') {
     var target = window.cordova ? '_system' : '_blank';
-    window.open(url, target);
+
+    //To detect if the application is running in electron or other browser window
+    function isElectron() {
+      // Renderer process
+      if (
+        typeof window !== 'undefined' &&
+        typeof window.process === 'object' &&
+        window.process.type === 'renderer'
+      ) {
+        return true;
+      }
+
+      // Main process
+      if (
+        typeof process !== 'undefined' &&
+        typeof process.versions === 'object' &&
+        !!process.versions.electron
+      ) {
+        return true;
+      }
+
+      // Detect the user agent when the `nodeIntegration` option is set to true
+      if (
+        typeof navigator === 'object' &&
+        typeof navigator.userAgent === 'string' &&
+        navigator.userAgent.indexOf('Electron') >= 0
+      ) {
+        return true;
+      }
+
+      return false;
+    }
```

Comment:
You can just do: 
```
const electron = this.getElectron();
if (electron) { ... }
```

Reply:
This is because getElectron method is the one responsible for getting the electron module. If it can't it will return null :) 
The rest of the codebase should not have to deal with complex code to known if we're inside Electron or not: just call this function ;) This way if we need to improve it later, there is a single place where we have to do this.